packages/edit-site/src/components/global-styles/font-library-modal/style.scss

```diff
@@ -36,19 +30,27 @@
 	}
 }
 
-.font-library-modal__fonts-grid {
-	.font-library-modal__fonts-grid__main {
-		display: flex;
-		flex-direction: column;
-	}
+.font-library-modal__fonts-grid__list {
+	margin: 0;
```

Comment:
I wonder if we could be more explicit and only override what we need here? I'm not a huge fan of shorthand for [these reasons](https://www.tempertemper.net/blog/why-i-prefer-not-to-use-css-shorthand). This is a nitpick and should not hold up this being merged.

```
margin-bottom: 0;
margin-top: 0;
```